src/ralph/settings/base.py

```diff
@@ -167,3 +168,7 @@
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',  # noqa
     'PAGE_SIZE': 10,
 }
+
+
+IMEI_UNTIL_2003 = re.compile(r'^\d{6} *\d{2} *\d{6} *\d$')
```

Comment:
We should think about another way for doing this - it's specific for allegro, so not open source applicable, nor should sits in settings in the current form.